docker/caddy/Caddyfile

```diff
@@ -0,0 +1,26 @@
+# Listen on all interfaces, default Caddy port
+:2015 {
+	# This is the pattern that will be used to detect
+	# WebSockets
+	@websockets {
+	    header Connection *Upgrade*
+		header Upgrade websocket
+    }
+
+	# Handle any traffic that matches the `@websockets` pattern
+	handle @websockets {
+		reverse_proxy http://endorser-agent:8052
+	}
+
+	# Handle all other traffic.
+	handle {
+		reverse_proxy http://endorser-agent:8050
```

Comment:
You can use environment variables rather than hardcoding the port numbers.

Reply:
Hmmm doesn't like env vars here ...
```
endorser-caddy_1        | {"level":"info","ts":1663779596.2097242,"msg":"using provided configuration","config_file":"/etc/caddy/Caddyfile","config_adapter":"caddyfile"}
endorser-caddy_1        | Error: adapting config using caddyfile: parsing caddyfile tokens for 'handle': /etc/caddy/Caddyfile:13 - Error during parsing: parsing caddyfile tokens for 'reverse_proxy': /etc/caddy/Caddyfile:12 - Error during parsing: due to parsing difficulties, placeholders are not allowed when an upstream address contains a scheme
```